tech-committee-charter.md

```diff
@@ -100,7 +100,13 @@ The TC follows a [Consensus Seeking](https://en.wikipedia.org/wiki/Consensus-see
 
 If an agenda item cannot reach a consensus a TC member can call for either a closing vote or a vote to table the issue to the next meeting. The call for a vote must be seconded by a majority of the TC or else the discussion will continue.
 
-For all votes, a simple majority of all TC members for, or against, the issue wins. A TC member may choose to participate in any vote through abstention.
+For all votes, a simple majority of a quorum of TC members for, or against, the issue wins. A TC member may choose to participate in any vote through abstention. A quorum is defined as > 1/2 of TC members voting for or against (members who abstain do not count towards the quorum). TC votes may be done synchronously or asynchronously. Asynchronous votes remain open for a period of time agreed upon by the TC when the issue is put to a vote. Aggregated TC vote results are shared publicly but the vote selections of individual members are not shared. The TC decides whether vote selections by members are anonymous or shared among members when the issue is put to a vote.
```

Comment:
> members who abstain do not count towards the quorum

What is the motivation for this? in the TC discussion the feeling was that maybe abstaining should still count as a form of a vote and be part of the quorum.

Reply:
That was just me misinterpreting the quorum [suggestion here](https://github.com/open-telemetry/community/pull/1547#discussion_r1230234126). I've pushed a commit which counts abstaining towards quorum.